website/versioned_docs/version-0.42/navigatorios.md

```diff
@@ -209,6 +209,8 @@ The default background color of the navigation bar.
 
 ### `interactivePopGestureEnabled`
 
+<!-- alex ignore retext-equality -->
```

Comment:
Can you change this and elsewhere to the `pop` rule? This seems to be ignoring _all_ rules.

Reply:
As in:

```
<!-- alex ignore pop -->
```

Reply:
@rickhanlonii 
fixed!